### DISCUSSION TITLE
Timeseries: way to force show axis

### DISCUSSION THREAD

**Discussion by simPod** [grafana/grafana#68712] [Dashboards, Panels & Visualizations]
I have few repeated Timeseries panels. Some contain series that is placed on second axis (right), some don't.

That results in me having some panels with second axis shown and some without it which results in different graph width.

For comparison

Graph panel:

![image](https://github.com/grafana/grafana/assets/327717/aacbc25b-c0f6-45d0-92c6-d0c91f0be561)

Timeseries panel:

![image](https://github.com/grafana/grafana/assets/327717/8c09b6fa-4e3f-477b-a2ef-7325cb68b78c)

I need a way to force enable the second axis so it's always present to pad the chart.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82673. Feel free to continue the discussion around this there. Thank you!
